src/main/java/de/zalando/aruha/nakadi/domain/Subscription.java

```diff
@@ -23,10 +29,15 @@
     @Size(min = 1, max = 1)
     private Set<String> eventTypes;
 
+    @NotNull
     private String consumerGroup = "none";
 
+    @NotNull
```

Comment:
This would force this field to be populated by clients. I think it should be managed by the persistence layer.